src/main/java/org/jitsi/jigasi/sounds/SoundNotificationManager.java

```diff
@@ -232,13 +233,30 @@ private CallContext getCallContext()
      */
     public void process(Presence presence)
     {
-        RecordingStatus rs = presence.getExtension(RecordingStatus.class);
-
-        if (rs != null
-            && gatewaySession.getFocusResourceAddr().equals(
-                presence.getFrom().getResourceOrEmpty().toString()))
+        if (gatewaySession.getFocusResourceAddr().equals(presence.getFrom().getResourceOrEmpty().toString()))
         {
-            notifyRecordingStatusChanged(rs.getRecordingMode(), rs.getStatus());
+            RecordingStatus rs = presence.getExtension(RecordingStatus.class);
+
+            if (rs != null)
+            {
+                notifyRecordingStatusChanged(rs.getRecordingMode(), rs.getStatus());
```

Comment:
I think this will result in playing "recording is off" when both jibri and audio recording are on and the jibri is stopped. I think we should read the recording state from both the jibri extension and ConferenceProperties together, and then act on it.